excelerator.md

```diff
@@ -2,3 +2,25 @@
 permalink: /excelerator.html
 ---
 
+# Excelerator  
+
+[http://excelerator.loci.cat](http://excelerator.loci.cat)
+
+## What 
+
+A data reapportionment tool. This tool will enable users to reapportion data referenced using one Loc-I data set to another Loc-I dataset, e.g. population count by ASGS Meshblocks reapportioned to Geofabric contracted catchments. 
+
+## Why 
+
+Reapportioning data is a frequently performed, time-consuming and error prone function. Pre-calculating the correspondences between Loc-I datasets and publishing these in machine-readable ways enable users to access and use reliable recipes for spatial reapportionment of data. 
```

Comment:
correspondences and ratios?

Reply:
i think 'correspondences' as they are aligning with the idea of https://www.abs.gov.au/websitedbs/d3310114.nsf/home/correspondences

Reply:
OK